core/spec/models/spree/concerns/display_money_spec.rb

```diff
@@ -16,7 +17,21 @@ def total
 
       context 'currency is not defined' do
         it 'generates a display_method that builds a Spree::Money without options' do
-          expect(test_class.new.display_total).to eq Spree::Money.new(10.0)
+          expect(test_instance.display_total).to eq Spree::Money.new(10.0)
+        end
```

Comment:
Please add new line.

```suggestion
        end
        
```